install_config/topics/configuring_for_azure_registry.adoc

```diff
@@ -0,0 +1,289 @@
+    --resource-group <resource-group> \
+    --output table
+
+KeyName    Permissions    Value
+key1       Full           <account-key>
+key2       Full           <extra-account-key>
+----
+
+Only one account key value is required for the configuration of the
+{product-title} registry.
+
+
+*Configuring the {product-title} registry for Azure using Ansible*
+
+.Procedure
+
+. Configure the Ansible inventory for the registry to use the recently created storage account:
+
+[source,yaml]
+----
+[OSEv3:vars]
+# Azure Registry Configuration
+openshift_hosted_registry_replicas=1 <1>
+openshift_hosted_registry_storage_kind=object
+openshift_hosted_registry_storage_azure_blob_accountkey=<account_key> <2>
+openshift_hosted_registry_storage_provider=azure_blob
+openshift_hosted_registry_storage_azure_blob_accountname=<account_name> <3>
+openshift_hosted_registry_storage_azure_blob_container=<registry_name> <4>
+openshift_hosted_registry_storage_azure_blob_realm=core.windows.net
+----
+<1> The number of replicas to configure.
+<2> The account key associated with the <account-name>.
+<3> The storage account name.
+<4> Directory used to store the data. `registry` by default
+
+
+*Manually configuring {product-title} registry for Microsoft Azure*
+
+To use Microsoft Azure object storage, edit the registry’s configuration file and mount to the registry pod.
+
+.Procedure
+
+. Export the current *_config.yml_*:
++
+[source,bash]
+----
+$ oc get secret registry-config \
+    -o jsonpath='{.data.config\.yml}' -n default | base64 -d \
+  >> config.yml.old
+----
+
+. Create a new configuration file from the old *_config.yml_*:
++
+[source,bash]
+----
+$ cp config.yml.old config.yml
+----
+
+. Edit the file to include the Azure parameters:
++
+[source,yaml]
+----
+storage:
+  delete:
+    enabled: true
+  cache:
+    blobdescriptor: inmemory
+  azure:
+    accountname: <account-name> <1>
+    accountkey: <account-key> <2>
+    container: registry <3>
+    realm: core.windows.net <4>
+----
+<1>	Replace with the storage account name.
+<2> The account key associated to the <account-name>.
+<3> Directory used to store the data. `registry` by default
+<4> Storage realm `core.windows.net` by default
+
+. Delete the `registry-config` secret:
++
+[source,bash]
+----
+$ oc delete secret registry-config -n default
+----
+
+. Recreate the secret to reference the updated configuration file:
++
+[source,bash]
+----
+$ oc create secret generic registry-config \
+    --from-file=config.yml -n default
+----
+
+. Redeploy the registry to read the updated configuration:
++
+[source,bash]
+----
+$ oc rollout latest docker-registry -n default
+----
+
+*Verifying the registry is using blob object storage*
+
+To verify if the registry is using Microsoft Azure blob storage:
+
+.Procedure
+
+. After a successful registry deployment, the registry `deploymentconfig` does
+not show any information if the registry is using an `emptydir` instead of Microsoft Azure blob storage:
++
```

Comment:
@bfallonf 
_...if the registry is using an `emptydir` instead of Microsoft Azure blob storage..._ Is this good or bad? 
If bad, how to fix?